packages/api-reference/docker/src/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the official Caddy image as a parent image
+FROM caddy:2-alpine
```

Comment:
I hope caddy alpine doesn't suffer from the same issues as node alpine 😅

Reply:
It does. Alpine linux is very unusual.